docs/what-is-this.adoc

```diff
@@ -28,7 +28,7 @@ and `maven-failsafe-plugin`.
 
 === Minimum requirements
 
-Java:: Requires Java 8.
+Java:: Requires Java 8. In case of using <<Affected>> strategy, Java 9 is not supported yet.
 
```

Comment:
I think that for the others it should work because we are not manipulating the `.class` but something to take a  look.